pkg/apis/extensions/v1beta1/types.go

```diff
@@ -640,3 +640,39 @@ type ClusterAutoscalerList struct {
 
 	Items []ClusterAutoscaler `json:"items"`
 }
+
+// A pod selector is a label query over a set of pods. The result of matchLabels and
+// matchExpressions are ANDed. An empty pod selector matches all objects. A null
+// pod selector matches no objects.
+type PodSelector struct {
+	// matchLabels is a map of {key,value} pairs. A single {key,value} in the matchLabels
+	// map is equivalent to an element of matchExpressions, whose key field is "key", the
+	// operator is "In", and the values array contains only "value". The requirements are ANDed.
+	MatchLabels map[string]string `json:"matchLabels,omitempty"`
+	// matchExpressions is a list of pod selector requirements. The requirements are ANDed.
+	MatchExpressions []PodSelectorRequirement `json:"matchExpressions,omitempty"`
+}
+
+// A pod selector requirement is a selector that contains values, a key and an operator that
```

Comment:
"a key, and an operator"
(though what you did is not wrong: https://en.wikipedia.org/wiki/Serial_comma)